<launch>
 <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear_lx" value="1" type="int"/>
  <param name="axis_linear_rx" value="3" type="int"/>
  <param name="axis_linear_y" value="2" type="int"/>
  <param name="axis_angular_z" value="0" type="int"/>
  <param name="scale_linear_lx" value="0.3" type="double"/>
  <param name="scale_linear_rx" value="0.5" type="double"/>
  <param name="scale_linear_y" value="0.3" type="double"/>
  <param name="scale_angular_z" value="0.3" type="double"/>
  <node pkg="ros_joy" type="teleop_joy" name="teleop"/>
</launch>
